api/v1alpha1/storageconsumer_types.go

```diff
@@ -39,9 +38,6 @@ const (
 
 // StorageConsumerSpec defines the desired state of StorageConsumer
 type StorageConsumerSpec struct {
-	// Capacity is the total quota size allocated to a consumer.
-	Capacity resource.Quantity `json:"capacity"`
```

Comment:
Are we going to have any problem with the upgrades if we remove an entry from CRD? @iamniting can you please confirm?

Reply:
I believe you will have an issue if any CR is using it. Usually, a breaking change needs to come with a version update where the k8s engine will translate from the prev version to the next one. 
But, as StorageConsumer is an internal API, and as we do not expect any upgrade from an older version, I think we are ok.

Reply:
Yes we are ok with this change.

Reply:
@Madhu-1 I have verified the upgrade path as well and it's working as expected.